client/extensions/woocommerce/state/wc-api/settings/general/reducer.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL,
+	WOOCOMMERCE_API_FETCH_SETTINGS_GENERAL_SUCCESS,
+} from '../../../action-types';
+
+export const LOADING = 'LOADING';
```

Comment:
I'm not a huge fan of `LOADING` here (and having a constant to load around various parts) but I'm having trouble thinking of what to use instead.